### ISSUE TITLE
Change name of runner to GZDoom

### ISSUE THREAD

**Issue by amethystgosling163** [lutris/lutris#5662]
The ZDoom runner actually installs GZDoom.

**Comment by strycore** [comment]
The Zdoom project uses Zdoom naming.

https://zdoom.org/index

**Comment by amethystgosling163** [comment]
> The Zdoom project uses Zdoom naming.
> 
> https://zdoom.org/index

Yes, that's right, but the runner name should reflect the real name of the runner so that the user knows that it is actually not the outdated ZDoom but its successor - GZDoom.

**Comment by amethystgosling163** [comment]
> The Zdoom project uses Zdoom naming.
> 
> https://zdoom.org/index

> The Zdoom project uses Zdoom naming.
> 
> https://zdoom.org/index

Yes, that's right, but the runner name should reflect the real name of the runner not the project name so that the user knows from the name Lutris doesn't actually use the outdated ZDoom but its successor - GZDoom.

**Comment by testman42** [comment]
I agree, there should be some indication that runner is GZDoom.
Because otherwise user is under impression that they are about to install an engine that [stopped being developed 9 years ago](https://doomwiki.org/wiki/ZDoom).
